mule-packager/src/main/java/org/mule/tools/api/packager/sources/MuleContentGenerator.java

```diff
@@ -106,10 +106,22 @@ public void createMetaInfMuleSourceFolderContent() throws IOException {
    *
    * @param classLoaderModel the classloader model of the application being packaged
    */
+  @Deprecated
   public void createApplicationClassLoaderModelJsonFile(ClassLoaderModel classLoaderModel) {
+    createApplicationClassLoaderModelJsonFile(classLoaderModel, true);
+  }
+
+  /**
+   * It creates classloader-model.json in META-INF/mule-artifact
+   *
+   * @param classLoaderModel the classloader model of the application being packaged
+   * @param prettyPrinting
+   */
+  @Deprecated
```

Comment:
This new one shouldn't be marked as deprecated.

Reply:
good catch! I will fix it